src/Model/TroopManager.php

```diff
@@ -57,4 +57,16 @@ public function deleteAll()
         }
         return "";
     }
+
+    /**
+     *  SQL query allowing to select the force in the troop table and to check if it can be executed.
+     */
+    public function selectTroop()
+    {
+        $select = $this->pdo->prepare("SELECT strength FROM " . self::TABLE . " WHERE id=:id");
+        if ($select->execute()) {
+            return $select->fetch(PDO::FETCH_ASSOC);
+        }
+        return "";
+    }
```

Comment:
You have a parameters that you don't assign a value to.
Don't return empty string without purpose.
And could you reuse the selectOneById() indead ?